src/content/kubernetes/best-practices/index.md

```diff
@@ -35,6 +35,17 @@ You don't want your development environment to be down, because of a minor node
 
 For production usage the cluster should have at least 5 nodes and a buffer of 2 nodes.
 
+#### Master nodes
+
+Master nodes matter much more than worker nodes in order to achieve better availability and performance. Several worker nodes can be replaced at the same time with no downtime but the failure of a master node could cause a major issue if it is your only master node.
```

Comment:
Looks like it would make sense to mention [HA masters](https://docs.giantswarm.io/advanced/high-availability/masters/) from this paragraph.

Reply:
Added HA info